automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -xeuE
+
+# This script should be able to execute multus
```

Comment:
if you want, add the license please, depends what is the de facto standard on this repo

Reply:
licence header is not used throughout the repo, if we want to add one, we should should discuss it. if decided that we want it, should be added in a separate PR for all scripts&code IMO